docs/changelog/index.html

```diff
@@ -0,0 +1,164 @@
+                        <label for="package-name-input">Package Name:</label>
+                        <!-- <input type="text" id="package-name-input" class="full-width"> -->
+                        <select id="package-name-input" class="full-width">
+                            <!-- Options will be populated here -->
+                        </select>
+                    </div>
+                    <div class="form-group" id="version-input-group">
+                        <label for="version-input">Version:</label>
+                        <input type="text" id="version-input" class="full-width">
+                        <span style="color:red" id="version-input-error"></span>
+                    </div>
+                    <div class="form-group" id="commit-message-group">
+                        <label for="commit-message-input">Commit Message:</label>
+                        <input type="text" id="commit-message-input" class="full-width">
+                    </div>
+                    <div class="form-group" id="commit-hash-group">
+                        <label for="commit-hash-input">Commit Hash:</label>
+                        <input type="text" id="commit-hash-input" class="full-width">
+                    </div>
+                </div>
+                <div class="form-row">
+                    <div class="form-group">
+                        <button type="submit" id="search-button" class="btn btn-primary">Search</button>
+                        <a id="reset-button" href="./" class="btn btn-default">Reset</a>
+                    </div>
+                </div>
+            </form>
+        </div>
+        <div class="results-container" id="search-results">
+            
+        </div>            
+      </div>
+      <div id="scrollTop">^</div>
+    </main>
+    <footer>
+      <div class="container">
+        <div class="row">
+          <div class="col-xs-12 col-md-12">
+            <div class="copyright">
+              &copy; 2015-2022 Cisco and/or its affiliates. All Rights Reserved.
+            </div>
+          </div>
+        </div>
+      </div>
+    </footer>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/handlebars@4.7.8/dist/handlebars.js"></script>
+    <script id="changelog-item-template" type="text/x-handlebars-template">
+      {{#each data.search_results}}
+        <div class="changelog-item">
+          <h3>Changelog for <span id="package-name">{{this.package}}</span>@<span id="version">{{this.version}}</span>:</h3>
+
+          <section class="commits">
+            <b>Commits: </b>
+            <ul>
+              {{#each (forIn this.commits)}}
+                <li>{{{github_linking this.value 'message'}}} (<span class="commit-hash">{{{github_linking this.key 'hash'}}}</span>)</li>
+              {{/each}}
+            </ul>
+          </section>
+
+          <section class="related-packages">
+            <b>These packages were also released along with this version:</b>
```

Comment:
```suggestion
            <b>These packages were also released along with the selected version:</b>
```

Reply:
They might sometimes not select a version but rather search with commit messages. This suggestion would look different at those times